docs/sql_reference/src/asciidoc/_chapters/sql_statements.adoc

```diff
@@ -4816,59 +4821,67 @@ grantor is:
 
 * `_privilege_ [,_privilege_ ] &#8230; | all [privileges]`
 +
-specifies the privileges to grant. you can specify these privileges for an object.
+Specifies the privileges to grant. You can specify these privileges for an object.
 +
 [cols="25%,75%"]
 |===
-| select     | can use the select statement.
-| delete     | can use the delete statement.
-| insert     | can use the insert statement.
-| references | can create constraints that reference the object.
-| update     | can use the update statement on table objects.
-| execute    | can execute a stored procedure using a call statement or can execute a user-defined function (UDF).
-| usage      | can access a library using the create procedure or create function statement. this privilege provides you 
-with read access to the library’s underlying library file.
-| all        | all the applicable privileges. when you specify all for a table or view, this includes the select, delete, 
-insert, references, and update privileges. when the object is a stored procedure or user-defined function (UDF), only the 
-execute privilege is applied. when the object is a library, only the update and usage privileges are applied.
+| DELETE                   | Can use the delete statement.
+| EXECUTE                  | Can execute a stored procedure using a call statement or can execute a user-defined function (UDF).
+| INSERT     [column-list] | Can use the insert statement.
+| REFERENCES [column-list] | Can create constraints that reference the object.
+| SELECT     [column-list] | Can use the select statement.
+| UPDATE     [column-list] | Can use the update statement on table objects.
+| USAGE                    | For libraries, can access a library using the create procedure or create 
+function statement. This privilege provides you with read access to the library’s underlying library 
+file. For sequences, can use the sequence in a SQL statement.
+| ALL                      | All the applicable privileges. When you specify all for a table or view, 
+this includes the select, delete, insert, references, and update privileges. When the object is a 
+stored procedure or user-defined function (UDF), only the execute privilege is applied. When the 
+object is a library, only the update and usage privileges are applied. When the object is a sequence
+generator, only the usage privilege is applied.
 |===
 
-* `on [_object-type_] [_schema_.]_object_`
+* `ON [_object-type_] [_schema_.]_object_`
 +
-specifies an object on which to grant privileges. _object-type_ can be:
+Specifies an object on which to grant privileges. _object-type_ can be:
 
-** `[table] [_schema_.]_object_`, where _object_ is a table or view. see <<database_object_names,database object names>>.
-** `[procedure] [_schema_.]_procedure-name_`, where _procedure-name_ is the name of a stored procedure in java (SPJ) 
+** `[FUNCTION] [_schema_.]_function-name_`, where _function-name_ is the name of a user-defined function (UDF) in the database.

```

Comment:
Not sure why the square brackets are needed around [FUNCTION] etc. They aren't optional are they? If the object_type is omitted, do we default to TABLE?